crypto/ct/ct_vfy.c

```diff
@@ -113,6 +113,10 @@ int SCT_CTX_verify(const SCT_CTX *sctx, const SCT *sct)
         CTerr(CT_F_SCT_CTX_VERIFY, CT_R_SCT_LOG_ID_MISMATCH);
         return 0;
     }
+    if (sct->timestamp > sctx->epoch_time_in_ms) {
```

Comment:
My thinking was that you'd set `sctx->epoch_time_in_ms` to take into account whatever clock drift you wanted to accommodate, e.g. set it 1 second into the future to allow up to 1 second of drift, rather than hard-coding a tolerance here. You'd do this via `CT_POLICY_EVAL_CTX_set_time`.

It depends where you're getting your SCT from. If you're submitting a new certificate to a log and immediately attempting to validate the SCT you get back, it's conceivable that the SCT would be less than 1 second old. One of the CT logs has been known to have clock drift that has caused it to issue SCTs with timestamps at least a minute in the future. Some people might want to accept that, others might want to reject it, so I'd rather not hard-code a value.

In the case where the SCTs are received while negotiating a TLS connection, the SCTs shouldn't be that fresh, so using `SSL_SESSION_get_time()` shouldn't be a problem.